src/DynamoCoreWpf/UI/GuidedTour/Guide.cs

```diff
@@ -143,7 +143,9 @@ internal void Play()
         /// </summary>
         internal void Initialize()
         {
-            TotalSteps = GuideSteps.Count;
+            //There are guided tours containing repeated Sequences due that we can have different Steps flow (conditional flows)
+            var differentSteps = GuideSteps.Select(step => step.Sequence).Distinct().ToList();
+            TotalSteps = differentSteps.Count;
```

Comment:
Will this change the total number of steps in each guided tour? For example if previously the package tour had 11 steps where 11 was the final step, so we calculated that if a user reached step 11, that means they completed the package tour, is it going to be a different number after this change?

Reply:
@RobertGlobant20 ^^

Reply:
@zeusongit yes, It will calculate the total of steps of each tour (I've tested the three of them).
Right now the Packages tour is counting 10 steps (as I was saying there are respeated Sequences for allowing to take a different path, for example if the Agreement has not been accepted a diferent Popup will be shown in that particular Step), if you see the json file we have only 10 steps including the Survey (this one starts from Sequence 1) https://github.com/DynamoDS/Dynamo/blob/1e79ba72e0fe334a17bcaded449a48ec8f4a990d/src/DynamoCoreWpf/UI/GuidedTour/dynamo_guides.json#L858